docs/website/content/integrations/opa-permissions-wrapper.md

```diff
@@ -0,0 +1,15 @@
+---
+title: OPA Permissions Wrapper
```

Comment:
I think the title and logo should be for Backstage, as that's what the integration is built for. "OPA Perissions Wrapper" isn't providing any clues as to what this is about.

Reply:
Good point :sweat_smile: